recovery/builddata.cpp

```diff
@@ -0,0 +1,170 @@
+#include "builddata.h"
+#include <QProcess>
+#include <QRegExp>
+#include <QFile>
+#include <QDebug>
+
+BuildData::BuildData()
+{
+}
+
+void BuildData::splitVersion(void)
```

Comment:
Probably makes sense to just move this functionality wholesale to inside `setVersion` ?

Reply:
I think I previously called splitVersion() from elsewhere, but didn't refactor it out in later iterations